DataRepo/templates/DataRepo/sample_list.html

```diff
@@ -31,12 +31,11 @@ <h4>List of Samples</h4>
                 <th data-filter-control="input" data-sortable="true" data-field="Tissue">Tissue</th>
                 <th data-filter-control="input" data-sortable="true" data-field="Studies">Studies</th>
                 <th data-filter-control="select" data-sortable="true" data-field="Genotype">Genotype</th>
-                <th data-filter-control="input" data-sortable="true" data-field="Tracer">Tracer</th>
+                <th data-filter-control="input" data-sortable="true" data-field="Infusate">Infusate</th>
                 <th data-filter-control="input" data-sortable="true" data-field="Treatment">Treatment</th>
-                <th data-filter-control="select" data-sortable="true" data-visible="false" data-field="Labeled_Atom">Labeled Atom</th>
-                <th data-filter-control="input" data-sortable="true" data-visible="false" data-field="Labeled_Count">Labeled Count</th>
+                <th data-filter-control="select" data-sortable="true" data-field="Labeled_Elements">Labeled Element(s)</th>
                 <th data-filter-control="input" data-sortable="true" data-visible="false" data-field="Infusion_Rate">Infusion Rate (ul/min/g)</th>
-                <th data-filter-control="input" data-sortable="true" data-visible="false" data-field="Infusion_Concentration">Infusion Concentration (mM)</th>
+                <th data-filter-control="input" data-sortable="true" data-visible="false" data-field="Concentrations">Infusion Concentration(s) (mM)</th>
```

Comment:
NB. I think this should be "Tracer Concentration(s)".
```suggestion
                <th data-filter-control="input" data-sortable="true" data-visible="false" data-field="Concentrations">Tracer Concentration(s) (mM)</th>
```

Reply:
fixed. Thanks!